src/test/java/com/amazon/opendistroforelasticsearch/security/OpendistroSecurityRolesTests.java

```diff
@@ -79,11 +80,15 @@ public void testOpenDistroSecurityRoles() throws Exception {
 		Assert.assertTrue(resc.getBody().contains("sr_user"));
 		Assert.assertTrue(resc.getBody().contains("xyz_sr"));
 
-		// Ensure hidden reserved and non-existent roles are not available
+		// Opendistro_security_roles cannot contain roles that don't exist.
 		Assert.assertFalse(resc.getBody().contains("xyz_sr_non_existent"));
-		Assert.assertFalse(resc.getBody().contains("xyz_sr_hidden"));
+
+		// Opendistro_security_roles can contain reserved roles.
 		Assert.assertTrue(resc.getBody().contains("xyz_sr_reserved"));
 
+		// Opendistro_security_roles cannot contain roles that are hidden in rolesmapping.yml.
+		Assert.assertTrue(resc.getBody().contains("xyz_sr_hidden"));
+
```

Comment:
Is this supposed to be True or False ?

Reply:
False, good catch. Was an error in the test + test config. I have fixed this.